Modules/Scripted/SampleData/SampleData.py

```diff
@@ -415,13 +415,20 @@ def registerCustomSampleDataSource(category='Custom',
       customDownloader=customDownloader,
       ))
 
+    
+  @staticmethod
+  def CustomDownloadTest(source):
+    raise Exception('CustomDownloadTesting')
```

Comment:
Use  `RuntimeError`, `Exception` is too general and the test could instead catch an unrelated exception

Reply:
`CustomDownloadTesting` -> `CustomDownloadTesting: This exception confirms that setting a custom download works`